scripts/rehype-prism.ts

```diff
@@ -0,0 +1,55 @@
+const visit = require('unist-util-visit');
```

Comment:
How come we need a custom syntax highlighting script?

Reply:
The rehype-prism library used previously did not have an option for loading the extra languages (like jsx / tsx) we needed. So I copied their file and added in the language loading piece. I will probably submit a PR back to the actual library but this was faster and I needed to test it anyways.